src/main/java/seedu/address/logic/commands/ResumeEditCommand.java

```diff
@@ -51,16 +56,38 @@ public CommandResult execute(Model model) throws CommandException {
 
         checkIndicesValidity(model);
```

Comment:
Do we want to block user if they accidentally key in an invalid index ?
Maybe we let it go through but give a warning?

Reply:
Hmm, good question. In terms of implementation, letting it through will then there will need to be some distinction in terms of feedbackToUser.

I am currently in favour of blocking them entirely, because it gives some space for the user to verify that what they are keying in is really what they want.

I’m open to other suggestions!